docs/book/appendix-editor-support.md

```diff
@@ -1,10 +1,25 @@
 # Appendix: Editor Support
 
-We have written a Visual Studio Code (VS Code) plugin that supports syntax highlighting for files written in the Litexa
+We have written a [Visual Studio Code (VS Code) extension](https://marketplace.visualstudio.com/items?itemName=litexa.litexa-syntax-highlighting) that supports syntax highlighting for files written in the Litexa
 language for Alexa skills. Any such files will be denoted with the `.litexa` file extension.
 
 ## Installation
 
+Visual Studio Code will prompt you to install the extension whenever you open a `.litexa` file. The dialog will look like this:
+
+![Visual Studio Code Install Prompt](../assets/images/vscode-litexa-extension.png)
+
+1. Click on "Search Marketplace"
+1. It should find the extension called "Litexa Syntax Highlighting", click "Install"
+1. That's it!
+
+### Alternative Install
+
+You can also search the Visual Studio Marketplace within Visual Studio Code for `ext:litexa` or `litexa` and install from there. More info about the extension can be found here:
+[https://marketplace.visualstudio.com/items?itemName=litexa.litexa-syntax-highlighting](https://marketplace.visualstudio.com/items?itemName=litexa.litexa-syntax-highlighting)
```

Comment:
Would potentially just add this as a footnote to the above "Installation", as opposed to having the separate header.

`Alternatively, you can also search the Visual Studio Marketplace ...`